src/components/GetReviews.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useContext, useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import { getReviews } from "../api";
+
+export default function GetReviews() {
+  const [isLoading, setIsLoading] = useState(true);
+  const [reviews, setReviews] = useState([]);
+
+  useEffect(() => {
+    // setIsLoading(true);
+    getReviews().then((allReviews) => {
+      setReviews(allReviews.results);
+      setIsLoading(false);
+    });
+  }, []);
+
+  if (isLoading) {
+    return <h3>Loading content...</h3>;
+  } else {
+    return (
+      <div id="reviewContaining">
+        {reviews.map((review) => {

```

Comment:
I like your date formatting here - consider extracting to keep the component cleaner